examples/projection_timing.py

```diff
@@ -0,0 +1,122 @@
+from networkx.algorithms import bipartite
```

Comment:
While we certainly appreciate the effort put into demonstrating the performance improvement, NetworkX currently doesn't have a natural place for benchmarks to exist. In practice I think we'd want not to include the benchmark at all.